src/controllers/customer.js

```diff
@@ -48,6 +48,25 @@ const createCustomer = async (req, res) => {
     }
 }
 
+const updateCustomer = async (req, res) => {
+    const id = Number(req.params.id)
+    const { name } = req.body
+
+    const customer = await prisma.customer.update({
+        where: {
+            id: id
+        },
+        data: {
+            name,
+        },
+        include: {
```

Comment:
See I included all of the fields in the object to be true where you just included the contacts... Probably the better way of doing it